bodhi/tests/functional/test_updates.py

```diff
@@ -1766,8 +1802,8 @@ def test_edit_update_bugs(self, publish, *args):
         bug_ids = [bug['bug_id'] for bug in up['bugs']]
         self.assertIn(56789, bug_ids)
         self.assertIn(98765, bug_ids)
-        self.assertEquals(up['status'], u'testing')
-        self.assertEquals(up['request'], None)
+        self.assertEquals(up['status'], u'pending')
+        self.assertEquals(up['request'], u'testing')
```

Comment:
Excellent!   This was my question, and I'm glad it's answered by the test suite.